streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1645,6 +1606,7 @@ private Consumer<StreamThread> streamThreadLeaveConsumerGroup(final long remaini
             if (groupInstanceId.isPresent()) {
                 log.debug("Sending leave group trigger to removing instance from consumer group: {}.",
                     groupInstanceId.get());
+                System.err.println(groupInstanceId.get());
```

Comment:
leftover debugging code I'm guessing? 😄

Reply:
> leftover debugging code I'm guessing? 😄

Ahhhhhh... looks like someone left their debugging breadcrumbs 😬 

![image](https://github.com/user-attachments/assets/65a02f40-aa08-4ac0-8f16-292a8a644259)

Reply:
Back to the topic.
@ableegoldman 
I have some thoughts about removing this method: https://github.com/apache/kafka/pull/19400#discussion_r2062833316

Reply:
The test of close consumer we already have a test: `org.apache.kafka.clients.consumer.KafkaConsumerTest#testGracefulClose`

It would give us enough protection after we replace it with the new close API.
I would like to do it in a separate PR.
https://issues.apache.org/jira/browse/KAFKA-19249